pymagicc/api.py

```diff
@@ -89,7 +82,7 @@ def run(self, only=None):
         """
         command = [join(self.run_dir, _magiccbinary)]
 
-        if not _WINDOWS and _magiccbinary.endswith(".exe"):
+        if not _WINDOWS and _magiccbinary.endswith(".exe"):  # pragma: no cover
```

Comment:
Is this because this statement is always true when testing under Travis?

Reply:
I added it to not count it a uncovered when checking Code Coverage under a non-WIndows system.